packages/scss/src/components/_menu.scss

```diff
@@ -79,7 +87,7 @@
 	}
 }
 
-.menu-link-placeholder { // legacy syntax 
+.menu-link-placeholder { // legacy syntax
```

Comment:
this code can be deleted it was to avoid movement between font-weight 400 and 600 but know we don't have this different font-weight on text. cc @vvalentin-lucca

Reply:
✅

Reply:
It becomes breaking change, doesn't it?

Reply:
@vvalentin-lucca so, I discussed it with @AlexiaAlarza and we are going to revert this section then remove all the legacy in a further release (in order to avoid having too much breakings on 8.2)

Reply:
And therefore put back the bolding?